cypress/integration/settings/create_new_parzelle_spec.js

```diff
@@ -1,15 +1,18 @@
 describe('Create new Parzelle/Allotment', function() {
-  before(function() {
-    cy.visit('/window/540210');
+  let name;
+  it('Read fixture and prepare the names', function() {
+    cy.fixture('settings/create_new_parzelle_spec.json').then(f => {
+      name = f['name'];
+    });
   });
-
-  it('open new parzelle', function() {
+  it('Open new parzelle', function() {
+    cy.visit('/window/540210');
     cy.clickHeaderNav('new');
   });
 
-  it('create it', function() {
-    cy.writeIntoStringField('Value', 'Industriestrasse');
-    cy.writeIntoStringField('Name', 'Industriestrasse');
+  it('Create it', function() {
+    cy.writeIntoStringField('Value', name);
+    cy.writeIntoStringField('Name', name);
     cy.get(':nth-child(4) > .header-btn > .header-item-container > .header-item').click();
```

Comment:
This looks like arcane magics to me. What are we trying to click here? Can we use/make a function for this? (or is this a 1 time-only kind of thing). At least a comment on what we're clicking should be added if nothing else.

Reply:
to be honest I'm not sure exactly what's the concept of parzelle about, but as I read through the translations, an parzelle=allotment which means a plot of land rented by an individual for growing vegetables or flowers.. I've added some doc here, I hope it's a bit more understandable.

Reply:
I wasn't referring to "parzelle" but to this specific line: `cy.get(':nth-child(4) > .header-btn > .header-item-container > .header-item').click();`

Reply:
I have added a comment:     /*show newly created parzelle in the grid view*/

Reply:
Replaced with cy.visitWindow...